.github/workflows/ci.yml

```
@@ -0,0 +1,93 @@
+      - name: cargo test (doc tests)
+        run: cargo test --workspace --doc --locked
+
+  audit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4.1
+      - uses: dtolnay/rust-toolchain@v1
+        with:
+          toolchain: 1.81.0
+      - name: Dependency vulnerability audit
+        uses: actions-rs/audit-check@v1
```

actionlint (1.7.8)

the runner of "actions-rs/audit-check@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)